Flaky test

```
Compiling 10 files (.ex)
Generated sentry app
Wrote 65 files in 367.84 kb to: _build/test/lib/sentry/priv/sentry.map
Running ExUnit with seed: 652526, max_cases: 8
....................................................
  1) test discard threshold discards logged messages (Sentry.LoggerHandlerTest)
Error:      test/sentry/logger_handler_test.exs:64[9](https://github.com/getsentry/sentry-elixir/actions/runs/15250320368/job/42885656466?pr=892#step:9:10)
     Assertion failed, no matching message after [10](https://github.com/getsentry/sentry-elixir/actions/runs/15250320368/job/42885656466?pr=892#step:9:11)00ms
     The following variables were pinned:
       ref = #Reference<0.98[11](https://github.com/getsentry/sentry-elixir/actions/runs/15250320368/job/42885656466?pr=892#step:9:12)47606.2527854594.187007>
     The process mailbox is empty.
     code: assert_receive {^ref, %{message: %{formatted: "Second"}}}
     stacktrace:
       test/sentry/logger_handler_test.exs:656: (test)
     The following output was logged:
     
Error:      09:22:31.527 [error] First
     
Error:      09:22:31.528 [error] Second
     
Error:      09:22:31.528 [error] Third
     
...................................................................................................................................................................................................................................................................................................................
Finished in 6.3 seconds (2.3s async, 4.0s sync)
[14](https://github.com/getsentry/sentry-elixir/actions/runs/15250320368/job/42885656466?pr=892#step:9:15) doctests, 346 tests, 1 failure
Error: Process completed with exit code 2.
```


https://github.com/getsentry/sentry-elixir/actions/runs/15250320368/job/42885656466?pr=892#step:9:17

### Solution Brainstorm

TBD

--- a/test/sentry/client_test.exs
+++ b/test/sentry/client_test.exs
@@ -388,7 +388,7 @@ defmodule Sentry.ClientTest do
 
       event =
         fn ->
-          assert_receive {^ref, body}, 1000
+          assert_receive {^ref, body}, 5000
           assert [{%{"type" => "event"}, event}] = decode_envelope!(body)
           event
         end

--- a/test/sentry/logger_handler_test.exs
+++ b/test/sentry/logger_handler_test.exs
@@ -64,7 +64,7 @@ defmodule Sentry.LoggerHandlerTest do
 
     {:ok, _task_pid} = Task.start(fn -> raise "oops" end)
 
-    assert_receive {^ref, event}
+    assert_receive {^ref, event}, 5000
 
     assert event.user.user_id == 3
     assert event.extra.day_of_week == "Friday"

